roles/confluent.ksql/defaults/main.yml

```diff
@@ -11,6 +11,8 @@ ksql_java_args:
 ksql_custom_java_args: ""
 ksql_final_java_args: "{{ ksql_java_args + [ ksql_custom_java_args ] }}"
 
+ksql_rocksdb_path: ""
```

Comment:
default behavior should be for ROCKSDB_SHAREDLIB_DIR to get set. ksqldb team says /tmp should be fine for this